### ISSUE TITLE
module_collision test fail on MacOS

### ISSUE THREAD

**Issue by mobileink** [Octachron/codept#5]
The MacOS default file system is case-insensitive, so cloning the repo results in:

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'tests/complex/module_collision/A.mli'
  'tests/complex/module_collision/a.mli'
```

Obviously such collisions cannot occur on MacOS.  The only way I can think of to address this is something like:

```
  'tests/complex/module_collision/foo/A.mli'
  'tests/complex/module_collision/bar/a.mli'
```

but I don't know if that works for the test.

**Comment by Octachron** [comment]
This should be fixed on master: the collision now happens between a/a.mli and A.mli (after disabling the `-nested` option for this specific test).

**Comment by Octachron** [comment]
This is fixed in codept 0.11.0 .
